### ISSUE TITLE
PHP functions prefixed with a slash are being ignored

### ISSUE THREAD

**Issue by dgwyer** [php-gettext/Gettext#284]
I thought this issue was directly related to `wp-cli` and so I posted the original message there: https://github.com/wp-cli/i18n-command/issues/344

Basically when prefixing a PHP i18n with a slash it gets ignored.

e.g.

```php
<?php
// wp i18n make-pot . --domain=bar
\esc_html__('foo', 'bar'); // ignored
esc_html__('baz', 'bar'); // processed
```

**Comment by oscarotero** [comment]
This is in Gettext 4, right?

I'm not working on maintaining this version (not enough time) but accept PR. If you want to work on that, I guess this can be fixed somewhere here: https://github.com/php-gettext/Gettext/blob/4.x/src/Utils/PhpFunctionsScanner.php#L117

**Comment by oscarotero** [comment]
Fixed and released in [v4.8.8](https://github.com/php-gettext/Gettext/releases/tag/v4.8.8)
